RecoHI/HiMuonAlgos/python/HiRegitMuonDetachedTripletStep_cff.py

```diff
@@ -63,7 +63,7 @@
 
 # fitting: feed new-names
 hiRegitMuDetachedTripletStepTracks                 = RecoTracker.IterativeTracking.DetachedTripletStep_cff.detachedTripletStepTracks.clone(
-    AlgorithmName = cms.string('iter6'),
+    AlgorithmName = cms.string('undefAlgorithm'),
```

Comment:
Well, the muon regional tracks get merged with the standard global tracks.  We thought maybe the naming scheme should keep them independent, such that customers of standard tracking can tell them apart.  On the other hand, we don't set the 'tight' and 'highPurity' bits for the muon regional tracks, which should be used for any analysis.  If you think it's better that we use the standard naming convention we'll make the change.